first-task/src/components/Content.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useCallback, useEffect, useState } from 'react';
+
+export default function Content(){
+    const [albums, setAlbums] = useState([])
+    const [photos, setPhotos] = useState([])
+   
+     useEffect(() => {
+       fetch('https://jsonplaceholder.typicode.com/albums/').then((res) => {
+         return res.json();
+       }).then(data => setAlbums(data));
+     }, [])
+   
+     const albumClickHandler = useCallback((id)=> {
+       fetch(`https://jsonplaceholder.typicode.com/albums/${id}/photos`).then((res) => {
+         return res.json();
+       }).then(data => setPhotos(data));
+     })
+   
+     const backClickHandler = useCallback(()=>{
+       setPhotos([])
+     })
+   
+     return (
+         <div className="Content">
+           {photos.length === 0 && albums.map((item) => <div className="ContentInfo" onClick={() => albumClickHandler(item.id)}>{item.title}</div>)}
```

Comment:
you neet move value int onClick to hook `useCallback` because it needs to avoid rerender